### PR TITLE
Quality of life pull for my recent commits

### PR DESCRIPTION
**Pull request by Incoming5643**
RPGloot will no longer try to place scrolls in secret satchels, as this would slowly turn the drop table into these fairly useless joke items.

Goes through and genders most(all) the pets in the game so the gender change potion can work on them. Many previously hard defined generic animals can now be either male or female.

The secret satchel will make up to five attempts to insert a free satchel, though there will still only ever be one.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
improvement scrolls are fucking sick

File: code/modules/events/wizard/rpgloot.dm [hunk 1]
```diff
@@ -49,7 +49,7 @@
 	var/quality = target.force - initial(target.force)
 	if(quality > 9 && prob((quality - 9)*10))
 		user << "<span class='danger'>[target] catches fire!</span>"
-		if(target.burn_state == -1)
+		if(target.burn_state < 0)
```
Comment (ChangelingRain) [inline code/modules/events/wizard/rpgloot.dm]: use the burn state defines please

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 2]
```diff
@@ -56,6 +56,7 @@
 /mob/living/simple_animal/pet/cat/original
 	name = "Batsy"
 	desc = "The product of alien DNA and bored geneticists."
+	gender = MALE
```
Comment (Cheridan) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: Should Batsy be a male @KorPhaeron

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: Not canon

**Pull request merged**